.github/workflows/evmc.yml

```diff
@@ -22,9 +22,31 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
+      - name: Install cmake
+        run: |
+          sudo apt-get update -y
+          sudo apt-get upgrade -y
+          sudo apt-get install -y cmake
+          cmake --version
+
+      - name: Install necessary GLIBCXX version
+        run: |
+          strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
+          sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+          sudo apt-get update -y
+          sudo apt-get install -y gcc-9 g++-9
+          sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 90
+          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 90
+          g++ --version
+          gcc --version     
+          strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
+
       - name: Get dependencies
         run: |
           go get -v -t -d ./...
+          git config --global --add safe.directory /home/ia/go/src/github.com/ethereum/go-ethereum/tests/evm-benchmarks
```

Comment:
Is the full path to your `ia` directory correct?

Reply:
lol good catch, dunno how that worked...

Reply:
Fixed with https://github.com/etclabscore/core-geth/pull/508/commits/a66e4cedc0eda5e9b8d7df0b9a47a36382efa804